server/datastore/mysql/software_test.go

```diff
@@ -1064,3 +1225,33 @@ func testUpdateHostSoftware(t *testing.T, ds *Datastore) {
 	require.NoError(t, err)
 	validateSoftware(tup{"bar", lastYear}, tup{"baz", future}, tup{"qux", future})
 }
+
+func testListSoftwareByHostIDShort(t *testing.T, ds *Datastore) {
```

Comment:
Is this actually testing `ds.ListSoftwareByHostIDShort`?

Reply:
Fixed in a69b5a8e7